xbmc/video/dialogs/GUIDialogVideoManagerVersions.cpp

```diff
@@ -183,6 +184,10 @@ void CGUIDialogVideoManagerVersions::SetDefaultVideoVersion(const CFileItem& ver
   // update video details since we changed the video file for the item
   m_database.GetDetailsByTypeAndId(*m_videoAsset, itemType, dbId);
 
+  m_videoAsset->ClearArt();
+  CVideoThumbLoader thumbLoader;
+  thumbLoader.LoadItem(m_videoAsset.get());
+
   // notify all windows to update the file item
   CGUIMessage msg{GUI_MSG_NOTIFY_ALL,        0,           0, GUI_MSG_UPDATE_ITEM,
```

Comment:
This formatting required by clang / Jenkins. There has to be a way to tune this?